cfy_manager/components/restservice/db.py

```diff
@@ -147,6 +176,21 @@ def create_amqp_resources(configs=None):
     logger.notice('AMQP resources successfully created')
 
 
+@_connect_to_db
+def check_manager_in_table(connection):
+    try:
+        result = \
+            connection.execute(
+                "SELECT * FROM managers where hostname='{0}'".format(
+                    config[MANAGER][HOSTNAME])
+            )
+        return result.rowcount
+    except BaseException as err:
```

Comment:
BaseException catches a lot of scenarios (like a keyboard cancel )

Reply:
Fixed